app/helpers/logging_helper.rb

```diff
@@ -0,0 +1,34 @@
+module LoggingHelper
```

Comment:
@maxious this gem might help for keeping thread locals tidy across requests: http://stackoverflow.com/a/13930563/4570488